### PR TITLE
Graphics Team 2018Q4 report

### PR DESCRIPTION
**Pull request by zeising**
Hi!
Here is the Q4 report from the graphics team!

### CODE REVIEW

File: 2018q4/graphics.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+are build with wayland support enabled.
+This makes it much easier to use and test wayland.
+
+Support for VMware graphics pass through has been added to the kernel driver.
+Support for this is still missing in `graphcs/mesa-dri` though, so it currently
+does not work out of the box.
+
+The input stack has been updated and is now for the most part current with
+upstream.
+Evdev headers where split off from `multimedia/v4l_compat` into their own port,
```
Comment (debdrup) [inline 2018q4/graphics.md]: s/where/were/

File: 2018q4/graphics.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+something we are planning to do in early February.
+A remnant of the drm2 code will remain in the base after this due to an
+unresolved dependency for arm TEGRA.
+Plans for its migration are expected to be finalized in first quarter in 2019.
+
+Support for i386 and PowerPC 64 has been added to the drm kernel drivers.
+This is currently in an alpha state.
+
+Wayland has been enabled by default in the ports tree, meaning that all packages
+are build with wayland support enabled.
```
Comment (debdrup) [inline 2018q4/graphics.md]: This is just the first example, but please do s/wayland/Wayland/g to keep it consistent across the document.

**Comment by zeising** [comment]
> Please see the notes I made.

Thank you!
I'll get on it.

**Comment by zeising** [comment]
Fixes done.

**Comment by debdrup** [comment]
Excellent work! Merged, so closing #39

**Pull request merged**